### ISSUE TITLE
Research and decide approach to new office pages

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2896]
### Summary

We're going to move office pages currently linked off organization chart to transition pages to Wagtail. We'll need to decide what page template to use, or if we need to create a new one. 

### Proposed Behavior / Feature

Background issue and inventory of pages: #2876 and #2877 

[Inventory document](https://drive.google.com/open?id=1qakIfDL-ccq8o590xyfOpwn972DwyD9k)

We will want to move this type of content into its Wagtail page:
![image](https://user-images.githubusercontent.com/24437369/57806150-8f6c0c00-772c-11e9-9e38-8aee6e028369.png)

### To Do
- [x]  Choose our approach to making these pages: we need to choose whether to follow the model at https://www.fec.gov/office-inspector-general/ or the model of the Commissioner pages, or make a new model
- [x] Decide whether we want to make these pages child pages of "About" or child pages of the new organization chart to be designed. Do we need to work on the new org chart first?

### Use Case

The offices of the FEC would no longer have outdated pages that are hard to maintain in Contribute.

### Future work
-Make implementation ticket(s)
-Choose format for the new pages
-Draft content for each page
-Review by each office
-Create the new pages
-Second pair of eyes
-Publish the new pages
-Add link to them to organization chart (See #2877)
-Add to redirect list and put in redirects
-Take down old pages

**Comment by dorothyyeager** [comment]
Looking into setting up a meeting for next week to talk about this.

**Comment by dorothyyeager** [comment]
We're going to meet on 7/10 to make these decisions.

**Comment by dorothyyeager** [comment]
The front-end, content and U/X design teams had a good meeting and we've realized we need to start with the organization chart as an MVP - turning it into something that can be edited in Wagtail and resemble the rest of the website while retaining the chart's basic organizational layout.

We'll tackle the questions in this particular issue once we've got the chart moved along. So for now, we'll table the office pages work and continue to update them in transition. The focus now turns to the organization chart. Work on the chart will pick up in #2877 .
